rascal-vscode-extension/src/lsp/RascalLSPConnection.ts

```diff
@@ -71,13 +71,20 @@ export async function activateLanguageClient(
     return client;
 }
 
+const contentPanels: Map<string, vscode.WebviewPanel> = new Map();
+
 async function showContentPanel(url: string, title:string, viewColumn:integer): Promise<void> {
     // dispose of old panel in case it existed
     const externalURL = (await vscode.env.asExternalUri(vscode.Uri.parse(url))).toString();
-    const allOpenTabs = vscode.window.tabGroups.all.flatMap(tg => tg.tabs);
-    const tabsForThisPanel = allOpenTabs.filter(t => t.input instanceof vscode.TabInputWebview && t.label === externalURL);
-
-    await vscode.window.tabGroups.close(tabsForThisPanel);
+    const id = title;
+    const existingPanel = contentPanels.get(id);
+
+    if (existingPanel) {
+        // reuse the tab, but reload the content
+        existingPanel.reveal(viewColumn, true);
```

Comment:
what if the id is the same, but not the `viewColumn`? shouldn't we then cleanup the old panel at least?